be/src/http/http_client.h

```diff
@@ -93,7 +93,7 @@ class HttpClient {
     // used to get content length
     int64_t get_content_length() const {
         double cl = 0.0f;
-        curl_easy_getinfo(_curl, CURLINFO_CONTENT_LENGTH_DOWNLOAD, &cl);
+        curl_easy_getinfo(_curl, CURLINFO_CONTENT_LENGTH_DOWNLOAD_T, &cl);
```

Comment:
push @mofeiatwork to merge his PR for this: https://github.com/StarRocks/starrocks/pull/35818